enterprise_access/apps/customer_billing/management/commands/fetch_and_handle_stripe_events.py

```diff
@@ -0,0 +1,110 @@
+"""
+Management command to fetch and handle Stripe events.
+"""
+import logging
+from datetime import datetime
+
+import stripe
+from django.core.management.base import BaseCommand, CommandError
+
+from enterprise_access.apps.customer_billing.stripe_event_handlers import StripeEventHandler
+
+logging.getLogger('stripe').setLevel(logging.INFO)
+
+
+class Command(BaseCommand):
+    """
+    Command to fetch recent Stripe events and process them with StripeEventHandler.
+
+    This command can be used to:
+    - Fetch and process recent events of a specific type
+    - Process a specific number of events
+    - Handle missed events or reprocess existing ones
+    """
+    help = 'Fetch and handle recent Stripe events'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--event-type',
+            type=str,
+            help='Specific event type to fetch (e.g., "invoice.paid")',
+        )
+        parser.add_argument(
+            '--limit',
+            type=int,
+            default=10,
+            help='Number of events to fetch (default: 10, max: 100)',
+        )
+        parser.add_argument(
+            '--dry-run',
+            action='store_true',
+            help='Show events that would be processed without actually processing them',
+        )
+
+    def handle(self, *args, **options):
+        event_type = options.get('event_type')
+        limit = options['limit']
+        dry_run = options['dry_run']
+
+        # Validate limit
+        if limit < 1 or limit > 100:
+            raise CommandError('Limit must be between 1 and 100')
+
+        # Build parameters for Stripe API call
+        stripe_params = {'limit': limit}
+        if event_type:
+            stripe_params['type'] = event_type
```

Comment:
[nit] Maybe long term is to support the constraint of when the event was created to help narrow down when an event was emitted. 
https://docs.stripe.com/api/events/list#list_events-created
It could be useful given the 100 max event limit.

Reply:
good idea, lemme see if I can get the rolled up quick